app/client/components/dashboard/beneficiaryLiveFeed.jsx

```diff
@@ -69,13 +71,45 @@ class BeneficiaryLiveFeed extends React.Component {
                               </UserTime>
                             );
 
-                            if (transfer.transfer_type === 'PAYMENT') {
+                            let currency, fromExchangeTransfer, recipientCurrency;
+                            const transferAccountId = transfer.sender_transfer_account_id;
+                            if (transferAccountId) {
+                              const transferAccount = this.props.transferAccounts.byId[transferAccountId];
+                              currency = transferAccount && transferAccount.token && transferAccount.token.symbol;
+                            }
+                            const money = formatMoney(transfer.transfer_amount / 100, undefined, undefined, undefined, currency);
+
+
+                            if (transfer.from_exchange_to_transfer_id !== null && typeof(transfer.from_exchange_to_transfer_id) !== "undefined") {
+
+                              fromExchangeTransfer = this.props.creditTransfers.byId[transfer.from_exchange_to_transfer_id];
+                              const transferAccountId = fromExchangeTransfer.sender_transfer_account_id;
+                              if (transferAccountId) {
+                                const transferAccount = this.props.transferAccounts.byId[transferAccountId];
+                                recipientCurrency = transferAccount && transferAccount.token && transferAccount.token.symbol;
+                              }
+                              const transferToMoney = formatMoney(fromExchangeTransfer.transfer_amount / 100, undefined, undefined, undefined, recipientCurrency);
```

Comment:
per comment on schema, `fromExchangeTransfer. transfer_amount` is a bit confusing. thought it was the exchanged from amount. and then shouldn't it be `exchangedToMoney`?

Reply:
i think it's actually already backwards from the line above... so `transferToMoney` is correct, but `fromExchangeTransfer` should be `toExchangeTransfer`

Reply:
I'm confused now 😕

Reply:
comment i just tagged u in should explain it.